src/SmartComponents/BaselinesPage/BaselinesPage.js

```diff
@@ -39,10 +80,24 @@ export class BaselinesPage extends Component {
     }
 
     renderTable() {
+        const { baselineTableData, baselineListLoading, createBaselineModalOpened } = this.props;
+        const { columns } = this.state;
+
         return (
             <CardBody>
                 <div>
-                    <BaselinesTable kebab={ true } createButton={ true } exportButton={ true } hasSelect={ true }/>
+                    <BaselinesTable
+                        kebab={ true }
+                        createButton={ true }
+                        exportButton={ true }
+                        hasSelect={ true }
```

Comment:
For me, `hasSelect` can be a bit confusing given multiselect and single-select are both forms of selection.